File: src/main/kotlin/worker/models/appointments/MyAppointmentsResponse.kt
package worker.models.appointments

data class MyAppointmentsResponse(
        var appointments: ArrayList<AppointmentResponseObject> = arrayListOf(),
        var clinicians: ArrayList<GenericResponseObject> = arrayListOf(),
        var appointmentSessions: ArrayList<GenericResponseObject> = arrayListOf(),
        var locations: ArrayList<GenericResponseObject> = arrayListOf(),
        var cancellationReasons: ArrayList<GenericResponseObject> = arrayListOf()
)

File: NHSOnline.Backend.Worker/Areas/Appointments/Models/MyAppointmentsResponse.cs
using System.Collections.Generic;

namespace NHSOnline.Backend.Worker.Areas.Appointments.Models
{
    public class MyAppointmentsResponse
    {
        public IEnumerable<Appointment> Appointments { get; set; }
        public IEnumerable<Clinician> Clinicians { get; set; }
        public IEnumerable<AppointmentSession> AppointmentSessions { get; set; }
        public IEnumerable<Location> Locations { get; set; }
        public IEnumerable<CancellationReason> CancellationReasons { get; set; }
    }
}
